docs/guides/akamai/get-started/parent-child-accounts/index.md

```diff
@@ -0,0 +1,154 @@
+- *Child account users with limited access* - non-admin users of your account whose permissions are managed by the child account admin.
+- *Child account parent user* - a single user representing all Akamai partner users who have access to this child account. It’s used to manage your child account. Depending on the access level and permissions provided by the child account admin, it may be a child account parent user with either full or limited access. To learn more, see [Manage account access and permissions for the child account parent user](#manage-account-access-and-permissions-for-the-child-account-parent-user).
+
+### Manage account access and permissions for the child account parent user
+
+By default, a child account parent user user has:
+- Read/write permission for billing. This is the only permission that can’t be modified even by a child account admin.
+- Read-only permission to all asset instances existing at the time of parent/child relationship creation, such as linodes, load balancers, longview clients. This allows the child account parent user to see existing child accounts’ assets, but not create, update, or delete them.
+- No access to asset instances created after the creation of the parent/child relationship. The child account parent user needs to ask a child account admin to grant them access to new instances if necessary.
+- The [global permissions](/docs/products/platform/accounts/guides/user-permissions/) set off.
+
+As a child account admin you can change the default permissions of the child account parent user to expand or restrict access as you see fit, except for billing. If you provide the child account parent user with full access to the child account, it becomes the child account parent user with full access and can fully manage your child account, including its users.
+
+To manage access settings:
+1. Log in to the [Cloud Manager](https://cloud.linode.com/).
+1. In the main menu, go to **Account**.
+1. In the **User & Grants** tab, in the **Parent User Settings** table, next to the name of the child account parent user, click **Manage Access**.
+1. In the **General Permissions** section:
+    1. Select the permissions you want to grant. Remember that permissions with the $ symbol next to them, may incur additional charges. Note also the **Full Account Access** switch that gives the child account parent user full access to your account.
+    1. Click **Save**.
+1. In the **Specific Permissions** section, select the access level for each feature or use the **Set all permissions to** dropdown. Click **Save**.
+
+### Delete a child account
+A child account admin can delete a child account only if the parent/child relationship is contractually removed, meaning you don’t have an active contract with an Akamai partner.
```

Comment:
I know that you're covering the actual _child account_ here, but do we want to call out the fact that a _child account parent user_ (proxy user) can't be deleted, somewhere in this section?